mmdet/models/backbones/resnet.py

```diff
@@ -352,6 +352,20 @@ class ResNet(nn.Module):
             memory while slowing down the training speed.
         zero_init_residual (bool): whether to use zero init for last norm layer
             in resblocks to let them behave as identity.
+
+    Example:
+        >>> from mmdet.models.backbones.resnet import *  # NOQA
```

Comment:
This may be more straightforward.
```python
from mmdet.models import ResNet
```

Reply:
Technically, this line isn't even needed because the `import *` is implicit when running a doctest, but I tend to include them so a user can copy-paste the code into an IPython terminal. 

However, I do like importing `ResNet` explicitly for clarity. How about `from mmdet.models.backbones.resnet import ResNet` so the import line references the exact file that it belongs to?

Ultimately, I'm happy with any option so tell me which one you like best: 

1. remove the line and let the import be implicit. 
2. `from mmdet.models import ResNet`
3. `from mmdet.models.backbones.resnet import ResNet`

Reply:
Yes an explicit import statement may be more clarified.
Although both 2 and 3 are acceptable to me, personally I prefer `from mmdet.models import ResNet` since it is clear enough and simpler.

Reply:
I did option 2.